pygef/test_files/cpt2.gef

```diff
@@ -0,0 +1,1136 @@
+#GEFID= 1, 1, 0
```

Comment:
Are we allowed to include this file?

Reply:
@breinbaas can we include the cpt file in the tests?

Reply:
Yes public data